.gitignore

```diff
@@ -7,3 +7,9 @@ ncp-debug.log
 npm-debug.log
 
 .env
+
+# Elastic Beanstalk Files
+.elasticbeanstalk/*
+!.elasticbeanstalk/config.yml
```

Comment:
ElasticBeanstalk automatically added everything except this line. It seems that it is not standard to commit the config.yml but for our usecase I think we need to.